Fabric.Authorization.API/web.config

```diff
@@ -26,7 +25,7 @@
       <remove name="WebDAV" />
       <add name="aspNetCore" path="*" verb="*" modules="AspNetCoreModule" resourceType="Unspecified" />
     </handlers>
-    <aspNetCore processPath="dotnet" arguments=".\Fabric.Authorization.API.dll" stdoutLogEnabled="true" stdoutLogFile=".\logs\stdout" >

```

Comment:
this seems like a weird change, was that intentional?

Reply:
No it wasn't, me and Brian were talking about it. It happened on mine and Jorden's using the new Access Control debug process. I'm removing it and creating new commits so it will be what it originally was. But it's something we should probably talk to Mike about.